src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1343,6 +1343,12 @@ private RendezvousAwaitable<int> InternalSendRequestAsync(WinHttpRequestState st
                     state.Dispose();
                     WinHttpException.ThrowExceptionUsingLastError();
                 }
+#if DEBUG
+                int lastError = Marshal.GetLastWin32Error();
+                Debug.Assert(!(unchecked((int)lastError) == Interop.WinHttp.ERROR_INSUFFICIENT_BUFFER ||
```

Comment:
Isn't this effectively dead code? If there was an error, wouldn't an exception have been thrown on the above line?